zarr/tests/test_storage.py

```diff
@@ -930,6 +940,31 @@ def test_pickle(self):
             pickle.dumps(store)
 
 
+@unittest.skipIf(pymongo is None, 'test requires pymongo')
+class TestMongoDBStore(StoreTests, unittest.TestCase):
+
+    def create_store(self):
+        # TODO: this is the default host for MongoDB on Travis,
+        # we probably want to generalize this though
+        store = MongoDBStore(host='127.0.0.1', database='zarr_tests',
+                             collection='zarr_tests')
+        # start with an empty store
+        store.clear()
+        return store
+
+
+@unittest.skipIf(redis is None, 'test requires redis')
+class TestRedisStore(StoreTests, unittest.TestCase):
+
+    def create_store(self):
+        # TODO: this is the default host for Redis on Travis,
```

Comment:
As above, is this TODO something that needs to be resolved before merge, or can we live with as-is?